browser/ui/webui/settings/brave_settings_localized_strings_provider.cc

```diff
@@ -265,6 +265,11 @@ void BraveAddCommonStrings(content::WebUIDataSource* html_source,
      IDS_SETTINGS_WEBRTC_POLICY_DEFAULT_PUBLIC_INTERFACE_ONLY},
     {"disableNonProxiedUdp",
      IDS_SETTINGS_WEBRTC_POLICY_DISABLE_NON_PROXIED_UDP},
+    {"requestOTRLabel", IDS_SETTINGS_REQUEST_OTR_LABEL},
```

Comment:
these are guarded by `enable_request_otr` in the grdp so I think they need to be here as well. @petemill what do we normally do for this stuff? Also use the buildflag here or don't use it in the grdp?

Reply:
added buildflag guard